sdk/inc/azure/core/az_result.h

```diff
@@ -60,13 +60,13 @@ enum
     } \
   } while (0)
 
-/**
- * The type represents success and error conditions.
- * Bits:
- * - 31 Severity (0 - success, 1 - failure).
- * - 16..30 Facility.
- * - 0..15 Code.
- */
+///
+/// The type represents success and error conditions.
+/// Bits:
+/// - 31 Severity (0 - success, 1 - failure).
+/// - 16..30 Facility.
+/// - 0..15 Code.
+///
```

Comment:
```suggestion
// The type represents success and error conditions.
// Bits:
// - 31 Severity (0 - success, 1 - failure).
// - 16..30 Facility.
// - 0..15 Code.
//
```

Reply:
"The type represents success and error conditions." should start with three `/`, the rest of the lines should have two.

Reply:
Why?

Reply:
So that "The type represents success and error conditions." goes to Doxygen, which is the good documentation about `az_result` enum. Otherwise, it will be without a description at all.

Reply:
Oh gotcha. In that case, we should use the consistent `/** @brief */` format for the doxygen docs, and `//` for source/implementation comments. Let's not mix in the `///` in there

Reply:
Given the docs change can go in whenever, we should focus on getting the source change in. We can always tweak the docs in a subsequent PR, post-preview 4.

Leaving the docs as is, reverting the docs change, or fixing it up, are all OK, imo.